zigpy_cli/radio.py

```diff
@@ -25,9 +25,10 @@
 @click.argument("radio", type=click.Choice(list(RADIO_TO_PACKAGE.keys())))
 @click.argument("port", type=str)
 @click.option("--baudrate", type=int, default=None)
+@click.option("--flow-control", type=int, default=None)
```

Comment:
Flow control isn't an integer, it should be a `click.Choice`. I believe I commented earlier a possible implementation.

Reply:
interestingly, the --help' was printing a 'string' so I didn't even spot that. I'll double check your previous comment! Sorry!

Reply:
Hmm, this actually doesn't work; maybe I did something wrong?

```
voluptuous.error.MultipleInvalid: value must be one of [None, 'hardware', 'software'] for dictionary value @ data['device']['flow_control']
```
Since I can't supply 'none' and apparently software is not none, this option is now mandatory?